test/PowerShellEditorServices.Test/Console/ChoicePromptHandlerTests.cs

```diff
@@ -85,6 +88,9 @@ public void ChoicePromptRepromptsOnInvalidInput()
             // Choice is invalid, should reprompt
             choicePromptHandler.ReturnInputString("INVALID");
 
+            // Give time for the prompt to reappear.
+            await Task.Delay(1000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4993607)